README.md

```diff
@@ -1,2 +1,42 @@
-# Leaderboard
-In this activity I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.
+# LeaderBoard
+In this activity you will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. 
+
+## Built With
+
+* JavaScript
+* HTML 
+* CSS
+* ES6
+* webpack
+
+### Getting Started
+For you to get a local template, please follow the folowing prerequisite steps:
+
+Access the live source on - [Live Demo](https://github.com/29td/Leaderboard/tree/leaderboard)
+Copy the **https** or **SSH** address on the code
+Clone the repository using your favorite code editor.
+Once you've open the repo on your VScode run it using index.html on src folder.
+
+## Author
+
+👤 **Reginah Nonhlanhla**
+
+- GitHub: [@Nonhlanhla](https://https://github.com/29td)
+- Twitter: [@NonhlanhlaMnde6](https://twitter.com/NonhlanhlaMnde6)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/29td/Leaderboard/issues).
+
+## Show your support
+
+Give a ⭐️ if you like our project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Big thanks to the microverse community with a special mention of my coding partner, morning session teams, standup teams and reviewers.
+## 📝 License
+This project is [MIT]() licensed.
```

Comment:
- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯